src/runmodes.c

```diff
@@ -157,6 +157,13 @@ static const char *RunModeTranslateModeToName(int runmode)
 #else
             return "WINDIVERT(DISABLED)";
 #endif
+        case RUNMODE_DPDK:
+#ifdef HAVE_DPDK
+            return "DPDK";
+#else
+            return "DPDK(DISABLED)";
```

Comment:
reword

Reply:
please check the right format of string to reply back. Are you enabling DPDK or enabling DPDK-Suricata?

Reply:
The function translates runmode id to the runmode name. Therefore if DPDK is enabled and the runmode id is equal to RUNMODE_DPDK then the string "DPDK" is returned. Otherwise if the runmode id is equal to RUNMODE_DPDK and DPDK is disabled then "DPDK(DISABLED)" is returned.